ParkingZoneApp/Areas/Admin/Controllers/ParkingZoneController.cs

```diff
@@ -6,156 +6,123 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Metadata.Internal;
 using ParkingZoneApp.Data;
 using ParkingZoneApp.Models;
+using ParkingZoneApp.Repositories;
 
-namespace ParkingZoneApp.Areas.Admin.Controllers
+namespace ParkingZoneApp.Areas.Admin
 {
     [Authorize]
     [Area("Admin")]
     public class ParkingZoneController : Controller
     {
-        private readonly ApplicationDbContext _context;
+        private readonly IGenericParkingZoneRepository<ParkingZone> _repository;
```

Comment:
https://github.com/Husniddin-coder/Parking-Zone/pull/9 see this PR and try to make like that